.github/workflows/check.yml

```diff
@@ -0,0 +1,27 @@
+name: Helm Chart CI
+
+on:
+  pull_request:
+
+jobs:
+  helm_chart_ci:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
❓ Should we use a TSCCR target for this?  I can't remember if that is the same for public repositories. same question for setup-helm action below.

Reply:
To my understanding TSCCR is an internal process and not used for public repositories. For example, in this recent PR https://github.com/hashicorp/terraform/pull/36023, no TSCCR verification action exists.